spec/generators/cucumber/install_generator_spec.rb

```diff
@@ -7,7 +7,16 @@
   # Tell the generator where to put its output (what it thinks of as Rails.root)
   destination File.expand_path('../../../../tmp', __dir__)
 
-  before { prepare_destination }
+  before do
+    prepare_destination
+    FileUtils.mkdir_p File.join(destination_root, 'config', 'environments')
+    %w[development test].each do |environment|
+      File.write(
+        File.join(destination_root, 'config', 'environments', "#{environment}.rb"),
+        "Rails.application.configure do\nend"
+      )
+    end
```

Comment:
Given the generator is now modifying existing configuration files, we'll need to ensure these files exist before exercising the system-under-test in the RSpec test suite.

Reply:
Cool, perhaps those new lines could have their own local helper method, with a name that captures your contextual comment?

Reply:
Excellent idea! Added in [`eca5bad`](https://github.com/cucumber/cucumber-rails/compare/884b75e689f15b7e8e335b397f3e586c00276392..eca5bad975acca459a3bf629888cbb8932ca59e2).